Python/Setup/swix/Packages/Microsoft.PythonTools.Profiling.Vsix.swixproj

```diff
@@ -1,16 +1,16 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Okay, I'd just suggest double-checking your output is exactly as you expect it to be then - I was  running into some file corruption when i did this earlier